greenline/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from uuid import uuid4
+
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Post(models.Model):
+    id = models.UUIDField(primary_key = True, default = uuid4, editable = False)
+    title = models.CharField(max_length = 200)
+    content = models.TextField(blank = True)
+    date_created = models.DateTimeField(auto_now_add = True)
+    date_modified = models.DateTimeField(auto_now = True)
+    author = models.TextField()
```

Comment:
Perhaps a nice way to spice this up from a generic post is to add additional properties, like maybe location if it's a travel blog post, ingredients/recipes if it's a cooking blog, or an address, postage cost, and send date for good ol' snail mail. Oh, the possibilities!

Reply:
Good thoughts.  I am envisioning a data visualization and data analysis blog, so maybe tags would be good.  I'll add that and anything else that seem pertinent.